mvc/src/main/java/com/interface21/webmvc/servlet/mvc/tobe/ClassInstance.java

```diff
@@ -0,0 +1,12 @@
+package com.interface21.webmvc.servlet.mvc.tobe;
+
+public class ClassInstance {
+
+    private Class<?> clazz;
+    private Object instance;
+
+    public ClassInstance(Class<?> clazz, Object instance) {
+        this.clazz = clazz;
+        this.instance = instance;
+    }
+}
```

Comment:
```suggestion
public class ClassInstance<T> {

    private final Class<T> clazz;
    private final T instance;

    public ClassInstance(Class<T> clazz, T instance) {
        this.clazz = clazz;
        this.instance = instance;
    }
}
```

ClassInstance 라는 클래스는 사용되고 있지 않은 듯 한데, 어떻게 사용하실 계획인가요?
기왕 사용을 한다면 generic 을 더 적극적으로 활용해보시기를 응원해봅니다.